### PR TITLE
fix(DIST-1089): Use important for width and height

### PR DESCRIPTION
**Pull request by scarciofolomarco**

### CODE REVIEW

**Review by mathio — APPROVED** [review]

**Comment by prilutskiy** [comment]
`!important` 😭 😭 😭

**Comment by prilutskiy** [comment]
Also, be extra careful with vw vh, Safari doesn't always like them.

**Comment by scarciofolomarco** [comment]
> `!important` 😭 😭 😭

yeah, I'm not proud but can't think a way to ignore `styles="width: #px; height: #px;"`

File: packages/embed/src/css/includes/_fullsize.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@mixin fullsize {
+  width: 100% !important;
+  height: 100% !important;
+  width: 100vw !important;
+  height: 100vh !important;
+}
```
Comment (prilutskiy) [inline packages/embed/src/css/includes/_fullsize.scss:6]: That is really good. We'll be able to switch to another solution once find it by updating just a single place in the code. Nice!

**Review by prilutskiy — APPROVED** [review]

**Pull request merged**